eradiate/tests/system/test_onedim_lambertian_brf.py

```diff
@@ -0,0 +1,119 @@
+"""Test cases with OneDimSolverApp and a lambertian surface."""
+
+import os
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+
+import eradiate
+
+matplotlib.rcParams["text.usetex"] = True
+
+eradiate_dir = os.environ["ERADIATE_DIR"]
+output_dir = os.path.join(eradiate_dir, "test_report", "generated")
+
+
+def ensure_output_dir(path):
+    if not os.path.isdir(path):
+        os.makedirs(path)
+
+
+@pytest.mark.slow
+def test_measured_lambertian_brf(mode_mono_double):
+    r"""
+    Measured lambertian BRF
+    =======================
+
+    This test case checks that measured BRF matches the Lambertian surface's
+    reflectance.
+
+    Rationale
+    ---------
+
+    * Geometry: a square surface with unit size and a Lambertian BRDF with
+      reflectance :math:`\rho \in [0.0, 0.3, 0.5, 0.7, 1.0]`.
+    * Illumination: Directional illumination with a zenith angle
+      :math:`\theta \in [0.0, 30.0, 60.0]°`.
+    * Sensor: Distant reflectance measure covering a plane (7 angular points,
+      :math:`10^5` samples per pixel) and targeting (0, 0, 0).
+
+    Expected behaviour
+    ------------------
+
+    The BRF results must agree with the reflectance input values within a
+    relative tolerance of 0.1%.
+
+    Results
+    -------
+
+    .. image:: generated/plots/test_measured_lambertian_brf_0.0.png
+       :width: 75%
+
+    .. image:: generated/plots/test_measured_lambertian_brf_30.0.png
+       :width: 75%
+
+    .. image:: generated/plots/test_measured_lambertian_brf_60.0.png
+       :width: 75%
+    """
+    spp = int(1e5)
+    n_vza = 7
+    zenith_values = [0.0, 30.0, 60.0]
+    reflectance_values = [0.0, 0.3, 0.5, 0.7, 1.0]
+
+    results = {}
+    for zenith in zenith_values:
```

Comment:
`zenith` is not explicit enough IMO, I suggest something like `illumination_zenith` for improved clarity.